ToDoList/src/ToDoList/ToDoList.sln

```diff
@@ -0,0 +1,31 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.4.33205.214
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ToDoList.Api", "ToDoList.Api\ToDoList.Api.csproj", "{3D701751-1608-4023-BCD5-7D0E5DB3EC1B}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ToDoList.Models", "ToDoList.Models\ToDoList.Models.csproj", "{C8A17443-D0BA-4F9A-84B9-1EF4D3BF0D12}"
```

Comment:
ToDoList.sln must be in root of repo.
src\ToDoList.Api.csproj
src\ToDoList.Models.csproj